packages/keychain/src/pages/session.tsx

```diff
@@ -151,11 +168,12 @@ export default function Session() {
 function sanitizeCallbackUrl(url: string): URL | undefined {
   try {
     const parsed = new URL(url);
+    const allowedHostnames = ["localhost"];
```

Comment:
Hmm will this prevent callbacks to i.e. the telegram app?

Reply:
This is only for callbacks so it wouldn't apply. Telegram uses redirects.